app/controllers/katello/api/v2/systems_controller.rb

```diff
@@ -61,6 +61,7 @@ def organization_id_keys
     param :organization_id, :number, :desc => N_("Specify the organization"), :required => true
     param :environment_id, String, :desc => N_("Filter by environment")
     param :host_collection_id, String, :desc => N_("Filter by host collection")
+    param :content_view_id, String, :desc => N_("Filter by content_ivew")
```

Comment:
Can you fix the description? I think it should just say "Filter by content view"